scripts/check.sh

```diff
@@ -20,8 +20,7 @@ check-protos-compatible() {
     export PATH=$GOPATH/bin:$PATH
 
     if [ ! -f "$GOPATH/bin/protolock" ]; then
-        GO111MODULE=off go get github.com/nilslice/protolock/cmd/protolock
-	    GO111MODULE=off go install github.com/nilslice/protolock/cmd/protolock
+	    GO111MODULE=off go install github.com/nilslice/protolock/cmd/protolock@latest
```

Comment:
Please specify protolock version to keep `proto.lock` consistent.

Reply:
https://github.com/pingcap/kvproto/pull/1224/commits/bf55f2e5756ba853432212cf2145bb80418b4203